elixir/Dockerfile

```diff
@@ -1,41 +1,29 @@
-ARG ALPINE_VERSION=3.18.2
-ARG OTP_VERSION=26.0.2
-ARG ELIXIR_VERSION=1.15.2
+ARG ALPINE_VERSION=3.18.4
+ARG OTP_VERSION=26.1.1
+ARG ELIXIR_VERSION=1.15.6
 
 ARG BUILDER_IMAGE="firezone/elixir:${ELIXIR_VERSION}-otp-${OTP_VERSION}"
 ARG RUNNER_IMAGE="alpine:${ALPINE_VERSION}"
 
 FROM ${BUILDER_IMAGE} as builder
 
-# install build dependencies
-RUN apk add nodejs npm build-base git python3
-
-# Add pnpm
-RUN npm i -g pnpm
-
 # prepare build dir
 WORKDIR /app
 
-# install hex + rebar
-RUN mix local.hex --force && \
-    mix local.rebar --force
-
 # install mix dependencies
 COPY mix.exs mix.lock ./
 COPY apps/domain/mix.exs ./apps/domain/mix.exs
 COPY apps/web/mix.exs ./apps/web/mix.exs
 COPY apps/api/mix.exs ./apps/api/mix.exs
-COPY config/ config/
```

Comment:
@jamilbk we must copy config, it can contain compile-time configuration for our dependencies (eg. for `mime` package) which will then fail to compile.

Reply:
Good catch, fixed!